### DISCUSSION TITLE
I know DOCX downloads are fiddly - but I need to know just *how* fiddly...

### DISCUSSION THREAD

**Discussion by psmedia** [knowledgefutures/pubpub#2873] [Support] [answered]
Hi, we're using PubPub to create the reference online version of a multichapter book publication (Infectious Disease Emergencies: Preparedness & Response, to be published by NUS Press). We've got around 120 authors and a great pool of collaborators volunteer assistant editors. The online edition will be the reference edition, and PubPub versioning is key here, but we will also publish a print edition of the book.

Given this "web centrality" I like the idea of using PubPub as a final stage editing tool - with authors and editors making "final manuscript" edits online. Then editors would download the MS Word document to send for typesetting for a print version (and eventual custom pdf download). Given the number of charts and graphics in this one, and the sensitivity of the full colour text layout, we are wedded to an InDesign based layout process and so need DOCX files. 

The problem I'm facing is that the DOCX downloads are fiddly. I've only tried three, but two of three arbitrarily stop before the end of the file. I understand that PanDoc conversions break easily, but I have two questions:

- how fiddly is fiddly? How often do they break? (OK, I know this is an impossible question, but any data points would be useful. Is what I'm seeing usual or abnormal).
- are there well-known points of failure for MS Word downloads? Tables formatted a certain way, images that break the column, number of citations, citation formats? Etc? 

I don't mind a certain number of failures, but if the majority of downloads fail then I need to design a process where MS Word files are manually updated with the changes made in the PubPub version (which means I have to publish before citations are complete, for example). You can see where I'd rather try and avoid that? 

Compare 

[95f9a6eb-94b5-4090-bd5e-b3eb3e9ee47a (2).docx](https://github.com/pubpub/pubpub/files/12906726/95f9a6eb-94b5-4090-bd5e-b3eb3e9ee47a.2.docx)

vs
https://emergencies.pubpub.org/pub/5-role-of-modelling/release/1?readingCollection=36ef2bb3 (still private)

**Comment by gabestein** [comment, marked as answer]
Hey -- thanks for using PubPub and writing out this question. Yes, it's a bit hard to answer exactly how fiddly they are, but I can say that they're generally pretty good and we have a few communities who use the output as part of their editorial process. Tables sometimes come through a little funky, but that's usually just formatting, not structural. Images, code, math, footnotes should all work fine. Citations, too (although it just formats them with superscript, I believe, since I don't think Pandoc output supports native Word citations or plugins like EndNote etc.)

My guess is the bug with the ones that are breaking partway through is actually related to a weird issue with some of the latest Desktop Office 365 programs and docx compatibility mode, which Pandoc outputs in. For example, the file you uploaded opens fine for me in Mac Pages, Google Docs, and Office 365 Online. 

When this happens, we've heard that folks can usually restore them by opening them and re-saving them without compatibility mode, then re-opening them. I've looked into this fairly extensively, and as far as I can tell it's a Word bug that has come and gone over the past 5 years during the transition to 365 -- but I'd certainly take pointers if someone else has a beat on this one.

**Reply by psmedia** [reply]
Well what do you know, that saving without compatibility works! I have access to the full file. Thanks @gabestein that's a big relief!
